railgun/Makefile

```diff
@@ -125,7 +125,20 @@ generate.controllers:
 clean.test.proxy:
 	helm uninstall --namespace kong-system kong-test-proxy
 
-# Our integration tests
+# Create only the base cluster that would be used for integration tests.
+# This can be helpful when developing new tests, as you can deploy the cluster
+# and run the test suite setup but then run tests individually against the cluster:
+#
+#   $ make test.integration.cluster KIND_CLUSTER_NAME="integration-tests"
+#   $ export KIND_CLUSTER="integration-tests"
+#   $ go test -v -run 'TestTCPIngress' ./test/integration/
+.PHONY: test.integration.cluster
+test.integration.cluster:
+	@./scripts/setup-integration-tests.sh
+	@go clean -testcache
+	@KIND_CLUSTER_NAME="$(KIND_CLUSTER_NAME)" KIND_KEEP_CLUSTER="true" GOFLAGS="-tags=integration_tests" go test -race -v -run "SuiteOnly" ./test/integration/
```

Comment:
```suggestion
	@KIND_CLUSTER_NAME="$(KIND_CLUSTER_NAME)" KIND_KEEP_CLUSTER="$(KIND_KEEP_CLUSTER)" GOFLAGS="-tags=integration_tests" go test -race -v -run "SuiteOnly" ./test/integration/
```

Reply:
double check me on https://github.com/Kong/kubernetes-ingress-controller/pull/1251#discussion_r621516761 about this, it's possible I'm missing something but I don't think we want to provide a default for this make variable?